src/ontology/mondo-edit.obo

```diff
@@ -486829,7 +486849,8 @@ property_value: curated_content_resource "https://www.malacards.org/card/ovarian
 
 [Term]
 id: MONDO:0020858
-name: mitochondrial complex 5 (ATP synthase) deficiency nuclear type 5
+name: mitochondrial complex V (ATP synthase) deficiency, nuclear type 5
+def: "Any mitochondrial complex deficiency in which the cause of the disease is a mutation in the ATP5F1D gene." []
```

Comment:
this definition needs an x-ref (maybe it can be "MONDO:patterns/disease_series_by_gene")